LoopKitUI/View Controllers/AddEditOverrideTableViewController.swift

```diff
@@ -0,0 +1,643 @@
+//
+//  AddEditOverrideTableViewController.swift
+//  Loop
+//
+//  Created by Michael Pangburn on 1/2/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import LoopKit
+
+
+public protocol AddEditOverrideTableViewControllerDelegate: AnyObject {
+    func addEditOverrideTableViewController(_ vc: AddEditOverrideTableViewController, didSavePreset preset: TemporaryScheduleOverridePreset)
+    func addEditOverrideTableViewController(_ vc: AddEditOverrideTableViewController, didSaveOverride override: TemporaryScheduleOverride)
+    func addEditOverrideTableViewController(_ vc: AddEditOverrideTableViewController, didCancelOverride override: TemporaryScheduleOverride)
+}
+
+// MARK: - Default Implementations
+extension AddEditOverrideTableViewControllerDelegate {
+    public func addEditOverrideTableViewController(_ vc: AddEditOverrideTableViewController, didSavePreset preset: TemporaryScheduleOverridePreset) { }
+    public func addEditOverrideTableViewController(_ vc: AddEditOverrideTableViewController, didSaveOverride override: TemporaryScheduleOverride) { }
+    public func addEditOverrideTableViewController(_ vc: AddEditOverrideTableViewController, didCancelOverride override: TemporaryScheduleOverride) { }
+}
+
+
+private extension TimeInterval {
+    static let defaultOverrideDuration: TimeInterval = .hours(1)
```

Comment:
Sometimes we stick constants in fileprivate extensions to take advantage of the concision of leading dot syntax. It could be a static member of `AddEditOverrideTableViewController`, but `.defaultOverrideDuration` reads more clearly than `AddEditOverrideTableViewController .defaultOverrideDuration` at sites of use.

Reply:
Ah, okay, I see why that would be nice. Okay, good reason!